### ISSUE TITLE
Enhancement: Migrate github to bitbucket

### ISSUE THREAD

**Issue by taylor** [jeffwidman/bitbucket-issue-migration#146]
It would be nice if this could go either direction. It seems like most of the functionality is there with the bitbucket to GH migration code.

**Comment by jeffwidman** [comment]
You're not wrong technically, but product wise there's no point as they're shutting down Bitbucket issues: https://github.com/jeffwidman/bitbucket-issue-migration/issues/148
